src/base/preferences.cpp

```diff
@@ -59,7 +59,7 @@
 
 Preferences *Preferences::m_instance = 0;
 
-Preferences::Preferences() {}
```

Comment:
Question only for my interest... Why is the new version (with default) better than the former (with empty body)?

Reply:
Just give more coding flexibility and less format violation options.